who-are-we.md

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
Need to include __frontmatter__ / `yaml` to any new topic-pages.  Otherwise, the `who-are-we.html` won't build for local dev-environments.

Reply:
Tested the following `yaml` locally.

```yaml
---
layout: default
---
```

![good-docs-pr51-review](https://user-images.githubusercontent.com/89339/93250099-454fcc00-f760-11ea-997a-17bc99b0b065.gif)